assets/src/domain/eventEditor/ui/ticketAssignmentsManager/data/useAssignmentManager.ts

```diff
@@ -93,6 +93,14 @@ const useAssignmentManager = (): AM => {
 				(assignmentType === 'forTicket' && entityType === 'tickets')
 			) {
 				relationalEntityToUse = pick([entity.id], relationalEntity);
+				// if it's for a new date or ticket
+				if (isEmpty(relationalEntityToUse)) {
+					const newRelationKey = entityType === 'datetimes' ? 'tickets' : 'datetimes';
+					// init to empty relations
+					relationalEntityToUse[entity.id] = {
```

Comment:
it's good that you used ramda at the source of this, it's immutable and safe :+1: